$NetBSD: patch-ac,v 1.1.1.1 2005/06/25 01:55:09 reed Exp $

make gvim default to using the right filetype syntax/ftplugin

--- icecc-2.9/tools.cpp.orig	2005-03-13 15:23:37.000000000 +0000
+++ icecc-2.9/tools.cpp	2005-03-13 15:24:21.000000000 +0000
@@ -33,7 +33,7 @@
 
 const char *tools[] = {
                         "icemc ~/.icewm/menu",
-                        "gvim ~/.icewm/preferences",
+                        "gvim \"+setf iceprefs\" \"+filetype plugin on\" ~/.icewm/preferences",
                         "icewoed ~/.icewm/winoptions",
                         "icemc ~/.icewm/toolbar",
                         "iceked ~/.icewm/keys",
